ember-app/app/models/new/issue.js

```diff
@@ -44,6 +44,7 @@ var Issue = Model.extend({
   },
   updateLabels : function () {
     this.set("processing", true);
+    this.set("labels", this.data.other_labels);
```

Comment:
This will only affect this specific issues labels array, the repo.labels are not touched - or am I missing something ?